.jules/friction/open/tokmd_context_git_clippy.md

```
@@ -0,0 +1,10 @@
+# `tokmd-context-git` fails workspace clippy checks
+
+## Description
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)